examples/js/loaders/GLTFLoader.js

```diff
@@ -1843,7 +1848,7 @@ THREE.GLTFLoader = ( function () {
 
 		return new Promise( function ( resolve, reject ) {
 
-			loader.load( resolveURL( bufferDef.uri, options.path ), resolve, undefined, function () {
+			loader.load( resolveURL( bufferDef.uri, options.path ), resolve, options.onProgressHandler, function () {
```

Comment:
I don't think it makes sense to use the same `onProgress` callback for multiple requests. Consider to use `LoadingManager` instead, see https://github.com/mrdoob/three.js/issues/15584#issuecomment-454515571

Reply:
LoadingManager works only with files, but onProgress in loader must return loading progress for current task.
I have gltf file about 30mb of binary bundle. Current implementation works fine, but loading manager return always 1.